packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -173,6 +173,15 @@ abstract class FlutterCommand extends Command<void> {
         'when running on remote machines.',
       hide: hide,
     );
+    argParser.addFlag('web-run-headless',
+      defaultsTo: false,
+      help: 'Launches the browser in headless mode.',
+      hide: true,
+    );
+    argParser.addOption('web-browser-debug-port',
+      help: 'The debug port the browser should use. If not specified, a random port is selected.',
```

Comment:
Specify it is only for chrome devtools

Reply:
Is it always going to be Chrome-only? How about "on browsers that support it"? This option is hidden anyway. The expectation is that the user knows what they are doing.

Reply:
Probably, I would still specify. This is documentation for the tools team too, someone who isn't experience on the web isn't going to known chrome devtools vs other browsers

Reply:
Done.